### ISSUE TITLE
The Khan scrapers are broken

### ISSUE THREAD

**Issue by jbampton** [fuchsia-programming/fuchsia-programming.github.io#205]
Looks like all the `khan` scrapers are broken :(

**Comment by mirelagrigoras** [comment]
Hello @jbampton. Is this the script that we should fix: https://github.com/slurpcode/slurp/blob/master/scrapers/python/lxml/khan.py?

**Comment by jbampton** [comment]
Hi @mirelagrigoras !!

Yes that is the `khan` Python script that needs fixing.  

But my Khan profile link is now -> https://www.khanacademy.org/profile/JohnBampton/

**Comment by jbampton** [comment]
I think the way the Khan pages are rendered has changed.

Seems they might be using dynamic JS to create the web pages.

**Comment by jbampton** [comment]
We need the `Energy points earned` data scraped. 

<img width="728" alt="Screen Shot 2020-08-30 at 11 26 51 pm" src="https://user-images.githubusercontent.com/418747/91660280-be241680-eb18-11ea-8a5a-a395b2a43b24.png">

**Comment by ajakov** [comment]
Yes, the way pages are rendered has changed. Whoever wants to fix other scrapers can check out the PHP scrapper, it works with the new layout and data fetching.

**Comment by jbampton** [comment]
Done in Python now

**Comment by wickedknock** [comment]
does it still needs fixing , really want to do some web scraping , if so please add me

**Comment by jbampton** [comment]
Yes they do need fixing
